### DISCUSSION TITLE
Duda respecto a donde escribir el código en la tarea

### DISCUSSION THREAD

**Discussion by BenjaCarrill0** [IIC2133-PUC/2024-1#9] [Tarea 0] [answered]
Buenos días, me surgió la duda si es que hay que programar todo en el archivo main.c que viene por defecto en el repositorio o si es que se puede modularizar el código.

Espero atentamente la respuesta.

**Comment by sachondo7** [comment, marked as answer]
Se puede modularizar sin ningun problema. Recuerda hacer los archivos .c y .h correspondientes

**Comment by ceojedab** [comment]
Hola, respecto a esto mismo, ¿es obligatorio modularizar en esta tarea o podemos escribir todo en el main.c? 
Muchas gracias.

**Reply by sachondo7** [reply]
No lo recomiendo pero si se puede, no hay descuentos por cantidad de líneas o cosas por ese estilo.
